src/formatDate.js

```diff
@@ -50,7 +50,53 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldDateSeparator = fromFormat.pop();
+  const newDateSeparator = toFormat.pop();
```

Comment:
In this task it doesn't really matter, but remember that `pop()` is a mutating method.
And in this case it mutates the function arguments.
You need to avoid this if it's not mentioned directly in the task description.